megamek/src/megamek/client/ui/swing/boardview/AbstractBoardViewOverlay.java

```diff
@@ -91,11 +74,9 @@ public AbstractBoardViewOverlay(Game game, ClientGUI cg, Font font, String heade
         currentPhase = game.getPhase();
         game.addGameListener(gameListener);
         clientGui = cg;
-        KeyBindParser.addPreferenceChangeListener(this);
-        GUIP.addPreferenceChangeListener(this);
 
         Color colorTitle = GUIP.getPlanetaryConditionsColorTitle();
-        header = String.format("#%02X%02X%02X %s",
+        header = String.format("#%02X%02X%02X%s",
                 colorTitle.getRed(), colorTitle.getGreen(), colorTitle.getBlue(), headerText);
```

Comment:
This can use the utility function AbstractBoardOverlay.colorToHex()

Reply:
Aye